### DISCUSSION TITLE
unplugin-vue-components support

### DISCUSSION THREAD

**Discussion by Bobakanoosh** [unovue/reka-ui#318] [Ideas]
Would be nice if this library maintained an [unplguin-vue-components resolver](https://github.com/antfu/unplugin-vue-components/blob/main/src/core/resolvers/_READ_BEFORE_CONTRIBUTE.md).

**Comment by zernonia** [comment]
That's a good suggestion @Bobakanoosh ! I have had good experience with `unplugin-vue-components`, and currently we have `autoImport` module for Nuxt. 

I believe this would improved the DX, and we can definitely add a resolver for autoImporting the components!

**Reply by zernonia** [reply]
https://github.com/radix-vue/radix-vue/issues/362

**Reply by zernonia** [reply]
This feature has been implemented 😁

**Reply by Bobakanoosh** [reply]
🎉 Awesome thank you so much!!
